src/components/ha-labeled-slider.ts

```diff
@@ -0,0 +1,96 @@
+import { CSSResultGroup, LitElement, css, html, nothing } from "lit";
+import { customElement, property } from "lit/decorators";
+import "./ha-icon";
+import "./ha-slider";
+import { fireEvent } from "../common/dom/fire_event";
+
+@customElement("ha-labeled-slider")
+class HaLabeledSlider extends LitElement {
+  @property() public labeled? = false;
+
+  @property() public caption?: string;
+
+  @property() public disabled?: boolean;
+
+  @property() public required?: boolean;
+
+  @property() public min: number = 0;
+
+  @property() public max: number = 100;
+
+  @property() public step: number = 1;
+
+  @property() public helper?: string;
+
+  @property() public extra = false;
+
+  @property() public icon?: string;
+
+  @property() public value?: number;
+
+  protected render() {
+    return html`
+      <div class="title">${this._getTitle()}</div>
+      <div class="extra-container"><slot name="extra"></slot></div>
+      <div class="slider-container">
+        ${this.icon ? html`<ha-icon icon=${this.icon}></ha-icon>` : nothing}
+        <ha-slider
+          .min=${this.min}
+          .max=${this.max}
+          .step=${this.step}
+          labeled=${this.labeled}
```

Comment:
```suggestion
          .labeled=${this.labeled}
```

Reply:
I think this one is incorrect. Labeled should be passed to ha-slider, it should be attached to the tag. 
```
<ha-slider {}></ha-slider> => no pin on the slider
<ha-slider labeled {}></ha-slider => pin on the slider
```